pkg/dockerregistry/client.go

```diff
@@ -19,10 +19,23 @@ import (
 	imageapi "github.com/openshift/origin/pkg/image/api"
 )
 
+const (
+	publicV1DockerHubHost = "index.docker.io"
+	// TODO: is there a better URL?
+	publicV2DockerHubHost = "registry-1.docker.io"
```

Comment:
Can't we just import `github.com/docker/docker/registry` and use `registry.DefaultV[12]Registry` constants?